R/model_info.R

```diff
@@ -29,7 +29,7 @@
 #' aps_lm_model4 <- lm(m2.price ~., data = apartments)
 #' model_info(aps_lm_model4)
 #'
-#' model_regr_rf <- randomForest::randomForest(m2.price~., data = apartments, ntree = 50)
+#' model_regr_rf <- ranger::ranger(m2.price~., data = apartments, num.trees = 50)
```

Comment:
in other examples there is library(ranger) function called before ranger() is called

Reply:
Done